cargo-workspaces/src/publish.rs

```diff
@@ -119,7 +139,19 @@ impl Publish {
                 IndexUrl::crates_io(None, None, None)?
             };
 
-            if is_published(&http_client, index_url, &name, version)? {
+            let already_published = is_published(&http_client, index_url, &name, version)?;
+
+            if self.only_print_plan {
+                let msg = if already_published {
```

Comment:
This makes the printing process longer, but in my use-case (manual workspace publishing) it was handy, as it acts as a reminder of where you stop.